<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>9</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="canvasGridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="coordLabel">
       <property name="text">
        <string>x=0, y=0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="saveToolButton">
       <property name="text">
        <string>Save ...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="closeToolButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeToolButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
